postUserAuthentication/impossibleTravelWorkflow.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  onPostAuthenticationEvent,
+  WorkflowSettings,
+  WorkflowTrigger,
+  getEnvironmentVariable,
+  createKindeAPI,
+  fetch,
+  denyAccess,
+} from "@kinde/infrastructure";
+
+// Workflow settings
+export const workflowSettings: WorkflowSettings = {
+  id: "impossibleTravelWorkflow",
+  name: "ImpossibleTravelCheck (TrustPath)",
+  failurePolicy: { action: "stop" },
+  trigger: WorkflowTrigger.PostAuthentication,
+  bindings: {
+    "kinde.auth": {},
+    "kinde.env": {},     // for env variables
+    "kinde.fetch": {},   // for API requests
+    "url": {},           // required
+  },
+};
+
```

Comment:
@ages96 thanks for raising this. If you look at the other examples there is generally a comment block here which explains what you need to do, i.e which env vars to set up. Could you add this?

Reply:
for example, explaining they need to setup `TRUSTPATH_API_KEY`